src/lib/pipeline.ml

```diff
@@ -45,11 +45,23 @@ let star_config =
 
 let vaxrank_config =
   let open Biokepi.Tools.Vaxrank.Configuration in
-  { default with
-    name = "PGV-configuration";
-    padding_around_mutation = 5;
-    max_vaccine_peptides_per_mutation = 3;
-    max_mutations_in_report = 20; }
+  {name = "PGV-configuration";
+   vaccine_peptide_length = 25;
+   padding_around_mutation = 5;
+   max_vaccine_peptides_per_mutation = 3;
+   max_mutations_in_report = 20;
+   min_mapping_quality = 1;
+   min_variant_sequence_coverage = 1;
+   min_alt_rna_reads = 3;
+   use_duplicate_reads = false;
+   drop_secondary_alignments = false;
+   mhc_epitope_lengths = [8; 9; 10; 11];
+   reviewers = [];
+   final_reviewer = "";
+   xlsx_report = true;
```

Comment:
They get translated to paths in the Biokepi code that uses these args (cf. https://github.com/hammerlab/biokepi/pull/388/files#diff-14f133a2f008d5a32a8cdec39363d489R168)